fv3core/stencils/ray_fast.py

```diff
@@ -36,13 +36,12 @@ def ray_fast_wind(
     pfull: FloatFieldK,
     dt: float,
     ptop: float,
+    rf_cutoff_nudge: float,
     ks: int,
     hydrostatic: bool,
 ):
     from __externals__ import local_ie, local_je, namelist
 
-    with computation(PARALLEL), interval(...):
-        rf_cutoff_nudge = namelist.rf_cutoff + min(100.0, 10.0 * ptop)
```

Comment:
oh yeah probably doesn't make sense to calculate a scalar in a stencil

Reply:
It was causing problems with validation as it was being promoted to a full 3D field.